package/batocera/core/batocera-configgen/configgen/configgen/generators/hypseus_singe/hypseusSingeGenerator.py

```diff
@@ -308,7 +308,7 @@ def copy_resources(source_dir: Path, destination_dir: Path) -> None:
 
         # Crosshair in supported games (e.g. ActionMax, ALG)
         # needCrosshair
-        if guns and system.config.get('singe_crosshair') != '0':
+        if system.config.use_guns and guns and not system.config.get('singe_crosshair'):
```

Comment:
Because we're getting the settings from an XML file, the values are going to be either `'0'` or `'1'`, so we want the following:

* unset - Add `-nocrosshair`
* off - Add `-nocrosshair`
* on - nothing

So we want the default to be the same as the "off" value. In this case, I'd recommend we use `get_bool()` instead of `get()` and give it a default of `True`:

```python
if system.config.use_guns and guns and system.config.get_bool('singe_crosshair', True):
```